app/views/system/admin/instance/users/index.html.slim

```diff
@@ -1,4 +1,5 @@
-= page_header
+= page_header do
+  = new_button(:admin_instance_user)
```

Comment:
The generated tooltip of `New Admin Instance user` isn't accurate. See the definition of `new_button` in the `application_widgets_helper.rb` and see if it can be modified.